Add `get_formatted_string()` convenience function (#241)
jazzband/prettytable: README.md

`print(x)`. You can now do all the usual things you can do with a string, like write
your table to a file or insert it into a GUI.

The table can be displayed in several different formats using `get_formatted_string`

> just by changing the `out_format=<text|html|json|csv|latex>`. This function passes

Reviewer: Thanks! Let's remove a couple of words, re: https://justsimply.dev/

```suggestion
by changing the `out_format=<text|html|json|csv|latex>`. This function passes
```

through arguments to the functions that render the table, so additional arguments

> can be given. The provides an easy means to let a user choose the output formatting.

Reviewer: ```suggestion
can be given. This provides a way to let a user choose the output formatting.
```
